pkg/limayaml/limayaml.go

```diff
@@ -7,51 +7,53 @@ import (
 )
 
 type LimaYAML struct {
-	MinimumLimaVersion    *string       `yaml:"minimumLimaVersion,omitempty" json:"minimumLimaVersion,omitempty"`
-	VMType                *VMType       `yaml:"vmType,omitempty" json:"vmType,omitempty"`
+	MinimumLimaVersion    *string       `yaml:"minimumLimaVersion,omitempty" json:"minimumLimaVersion,omitempty" jsonschema:"nullable"`
+	VMType                *VMType       `yaml:"vmType,omitempty" json:"vmType,omitempty" jsonschema:"nullable"`
 	VMOpts                VMOpts        `yaml:"vmOpts,omitempty" json:"vmOpts,omitempty"`
-	OS                    *OS           `yaml:"os,omitempty" json:"os,omitempty"`
-	Arch                  *Arch         `yaml:"arch,omitempty" json:"arch,omitempty"`
+	OS                    *OS           `yaml:"os,omitempty" json:"os,omitempty" jsonschema:"nullable"`
+	Arch                  *Arch         `yaml:"arch,omitempty" json:"arch,omitempty" jsonschema:"nullable"`
 	Images                []Image       `yaml:"images" json:"images"` // REQUIRED
-	CPUType               CPUType       `yaml:"cpuType,omitempty" json:"cpuType,omitempty"`
-	CPUs                  *int          `yaml:"cpus,omitempty" json:"cpus,omitempty"`
-	Memory                *string       `yaml:"memory,omitempty" json:"memory,omitempty"` // go-units.RAMInBytes
-	Disk                  *string       `yaml:"disk,omitempty" json:"disk,omitempty"`     // go-units.RAMInBytes
-	AdditionalDisks       []Disk        `yaml:"additionalDisks,omitempty" json:"additionalDisks,omitempty"`
+	CPUType               CPUType       `yaml:"cpuType,omitempty" json:"cpuType,omitempty" jsonschema:"nullable"`
+	CPUs                  *int          `yaml:"cpus,omitempty" json:"cpus,omitempty" jsonschema:"nullable"`
+	Memory                *string       `yaml:"memory,omitempty" json:"memory,omitempty" jsonschema:"nullable"`                   // go-units.RAMInBytes
+	Disk                  *string       `yaml:"disk,omitempty" json:"disk,omitempty" jsonschema:"nullable"`                       // go-units.RAMInBytes
+	AdditionalDisks       []Disk        `yaml:"additionalDisks,omitempty" json:"additionalDisks,omitempty" jsonschema:"nullable"` // commented out
```

Comment:
What does "commented out" mean?

Reply:
I think it meant to say "the values were commented out" (to motivate the need for a `null` array)

But it could be read as "the key should be commented out", and then it would **not** need to be null

Anyway, needs to be removed or clarified.

Thanks!